AntWars/GamePanel.cs

```diff
@@ -50,8 +50,11 @@ partial class GamePanel : Form {
         private void print(BoardObject[] boardObjects) {
             Bitmap tmp = new Bitmap(game.Conf.BoardWidth, game.Conf.BoardHeight);
 
-            for(int i = 0; i < boardObjects.Length;i++) {
-                setColor(boardObjects[i], tmp);
+            for (int i = 0;i < boardObjects.Length;i++) {
+                if (boardObjects[i] != null) {
```

Comment:
Wieso machst du Änderungen in meinem Branch?

Rein von der Theorie her darf das garnicht passieren.